src/components/atom_preference_language_checkbox/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { getLanguageFullName } from '@/global.constants'
+import { GlobalLanguageCode } from '@/global.interface'
+import { preferenceState } from '@/recoil/preferences/preference.state'
+import { Checkbox, FormControlLabel } from '@mui/material'
+import { FC, SyntheticEvent } from 'react'
+import { useRecoilCallback, useRecoilValue } from 'recoil'
+
+interface Props {
+  languageCode: GlobalLanguageCode
+}
+
+const PreferenceLanguageCheckbox: FC<Props> = ({ languageCode }) => {
+  const preference = useRecoilValue(preferenceState)
+  const onChange = useRecoilCallback(
+    ({ set, snapshot }) =>
+      async (e: SyntheticEvent<Element, Event>, checked: boolean) => {
```

Comment:
```suggestion
      async (_, checked: boolean) => {
```

the `e` (event) is no longer used